addons/addon-base-raas/packages/base-raas-cfn-templates/src/templates/service-catalog/emr-cluster.cfn.yml

```diff
@@ -181,6 +181,18 @@ Resources:
             Action:
               - 'sts:AssumeRole'
             Resource: 'arn:aws:iam::*:role/swb-*'
+          - Effect: Allow
+            Action:
+              - 'ssm:*'
+            Resource: '*'
+          - Effect: Allow
+            Action:
+              - 'ssmmessages:*'
+            Resource: '*'
+          - Effect: Allow
+            Action:
+              - 'ec2messages:*'
+            Resource: '*'
 
   Ec2Role:
```

Comment:
👍 The FAQ doc said to [add it to](https://w.amazon.com/bin/view/AWS/Solutions/SolutionsTeam/SolutionsImplementations/Service_Workbench_Technical_Resources/FAQ/#HPatchingwithSystemsManagerandWorkingwithPVRE) `IAmRole`, so I wasn't sure if this would also count.